doc/source/parameters/dem/insertion_info.rst

```diff
@@ -2,71 +2,76 @@
 
 
 .. code-block:: text
 
   subsection insertion info
-    # Insertion method
-    # Choices are uniform|non_uniform
+    # Choices are uniform|non_uniform|plane|list
     set insertion method                               = non_uniform
 
-    # Number of inserted particles at each insertion step. This value may change automatically if the insertion box is not adequately large to handle all the inserted particles
-    set inserted number of particles at each time step = 100
-
-    # Insertion frequency
+    # Every method
     set insertion frequency                            = 20000
 
-    # Insertion box dimensions (xmin, xmax, ymin, ymax, zmin, zmax)
+    # If method = uniform or non_unifom
+    set inserted number of particles at each time step = 100
+
     set insertion box minimum x                        = -0.05
     set insertion box minimum y                        = -0.05
     set insertion box minimum z                        = -0.03
     set insertion box maximum x                        = 0.05
     set insertion box maximum y                        = 0.05
     set insertion box maximum z                        = 0.07
 
-    # Directions of insertion (0: x, 1: y, 2: z)
     set insertion first direction                      = 0
     set insertion second direction                     = 1
     set insertion third direction                      = 2
 
-    # Initial conditions at insertion
     set velocity x                                     = 0.0
     set velocity y                                     = 0.0
     set velocity z                                     = 0.0
     set omega x                                        = 0.0
     set omega y                                        = 0.0
     set omega z                                        = 0.0
 
-    # insertion distance threshold controls the distance between the center of inserted particles (the distance is: [distance threshold] * [diameter of particles]). The distance is modified by a random number if non_uniform insertion is chosen
     set insertion distance threshold                   = 2
 
-    # Random number range and seed for non_uniform insertion
+    # If method = plane
+    set insertion method                               = plane
+    set insertion plane point                          = 0, 0, 0
+    set insertion plane normal vector                  = 1, 0, 0
+
+    # If method = non_uniform or plane
     set insertion random number range                  = 0.75
     set insertion random number seed                   = 19
-  end
 
-* The ``insertion method`` parameter chooses the type of insertion. Acceptable choices are ``uniform`` and ``non_uniform``.
+    # If method = list
+    set list x                                         = 0.
+    set list y                                         = 0.
+    set list z                                         = 0.
```

Comment:
Can you add the options for the velocity and angular velocity for the list insertion?

Reply:
I have look in the InsertionInfo::parse_parameters and in the ListInsertion<dim>::insert functions.

Velocity and angular velocity don't seems to be implemented for the list insertion method. I can you an other PR afterward. 
Maybe someone did implement this functionality in a branch, but it never got merge?